osbs/tekton.py

```diff
@@ -380,6 +380,11 @@ def start_pipeline_run(self):
             raise OsbsException("No input data provided for pipeline run to start")
 
         run_name = self.input_data.get('metadata', {}).get('name')
+        labels = self.input_data.get('metadata', {}).get('labels', {})
+
+        if labels:
+            sanitized_labels = {k: sanitize_strings_for_openshift(v) for k, v in labels.items()}
```

Comment:
`https://github.com/blah` is sanitized into `httpsgithub.comblah`. Is this what we need in the metadata.labels?

Reply:
well they were there before, we can think about removing some of them in the future